src/base/net/geoipmanager.cpp

```diff
@@ -124,8 +125,8 @@ void GeoIPManager::manageDatabaseUpdate()
 
 void GeoIPManager::downloadDatabaseFile()
 {
-    const QDate curDate = QDateTime::currentDateTimeUtc().date();
-    const QString curUrl = DATABASE_URL.arg(curDate.toString("yyyy-MM"));
+    const QLocale cLocale = QLocale("C");
+    const QString curUrl = DATABASE_URL.arg(cLocale.toString(QDateTime::currentDateTimeUtc(), "yyyy-MM"));
```

Comment:
Please change it to:
```c++
const QString curUrl = DATABASE_URL.arg(QLocale::c().toString(curDate, "yyyy-MM"));
```

Reply:
@Chocobo1 

Thanks, completely missed the existence of the `QLocale::c()` static member. Fixed.